### ISSUE TITLE
Developer doc does not match Bitcoin Core for locktime standard rule

### ISSUE THREAD

**Issue by daira** [bitcoin-dot-org/Bitcoin.org#3727]
https://github.com/bitcoin/bitcoin/blob/9faa4b68db80a4a5c1f5a4e0484d804d7b5e1493/src/consensus/tx_verify.cpp#L21 :
```
    if (tx.nLockTime == 0)
        return true;
    if ((int64_t)tx.nLockTime < ((int64_t)tx.nLockTime < LOCKTIME_THRESHOLD ? (int64_t)nBlockHeight : nBlockTime))
        return true;
```

https://developer.bitcoin.org/devguide/transactions.html#non-standard-transactions :
> As of Bitcoin Core 0.9.3, standard transactions must also meet the following conditions:
>
> * The transaction must be finalized: either its locktime must be in the past (or less than or equal to the current block height), or all of its sequence numbers must be 0xffffffff.

These do not match; "less than or equal to" should be "less than". Also it could be clearer that whether the restriction is time-based or height-based depends on whether the field is less than `LOCKTIME_THRESHOLD = 500000000`.

**Comment by murchandamus** [comment]
Yeah, I was just looking into why this was wrong again. As far as I can tell this was already fixed once in 2020 here: https://github.com/bitcoin-dot-org/Bitcoin.org/pull/3237. Did the code in this repo regress in some way since then?

**Comment by murchandamus** [comment]
Opened a PR to fix here: https://github.com/bitcoin-dot-org/developer.bitcoin.org/pull/244
